packages/components/src/components/tag/tag-shared-utils.ts

```diff
@@ -0,0 +1,45 @@
+import { getInsetJssStyle, ThemedColors } from '../../styles';
+import { JssStyle } from 'jss';
+import { TagColor } from './tag-utils';
+import { borderRadiusMedium, borderWidthBase, fontStyle, fontWeight } from '@porsche-design-system/utilities-v2';
+
+export const getTagFocusJssStyle = (themedColors: ThemedColors): JssStyle => {
+  return {
+    '&::before': {
+      content: '""',
+      position: 'absolute',
+      ...getInsetJssStyle(-4),
+      border: `${borderWidthBase} solid transparent`,
+      borderRadius: borderRadiusMedium,
+    },
+    '&:focus::before': {
+      borderColor: themedColors.focusColor,
+    },
+    '&:focus:not(:focus-visible)::before': {
+      borderColor: 'transparent',
+    },
+  };
+};
+
+export const getThemedBackgroundColor = (tagColor: TagColor, themedColors: ThemedColors): string => {
+  const colorMap: { [key in TagColor]: string } = {
+    'background-default': themedColors.backgroundColor,
+    'background-surface': themedColors.backgroundSurfaceColor,
+    'neutral-contrast-high': themedColors.primaryColor,
```

Comment:
FYI: As discussed with Janick and Johannes, they want the background color for 'neutral-contrast-high' to be primaryColor (black), because the 'contrastHighColor' is not dark enough

Reply:
Hmm, I see the point but I don't like the inconsistency. Why not adding in addition a new accepted color value `primary` and deprecate `neutral-contrast-high` while it's being mapped to `primary` for backwards compatibility reasons?

Same applies to `background-default`, which should be aliased to `background-base`.

Reply:
Done